atlasdb-console/versions.lock

```diff
@@ -338,13 +347,13 @@
             ]
         },
         "io.dropwizard:dropwizard-jackson": {
-            "locked": "0.9.3",
+            "locked": "1.0.6",
             "transitive": [
                 "com.palantir.atlasdb:atlasdb-config"
             ]
         },
         "io.dropwizard:dropwizard-util": {
-            "locked": "0.9.3",
+            "locked": "1.0.6",
```

Comment:
I am seeing how we can restrict only timelock-server to bump the dropwizard and jackson dependencies. It seems that many of these dependencies are pulled in transitively from other AtlasDB projects. I think we can exclude the transitive ones and add the updated versions only for timelock-server.